pkg/channel/distributed/controller/testing/data.go

```diff
@@ -165,6 +164,7 @@ Producer:
   Idempotent: true
   RequiredAcks: -1
 `
+	ConfigMapVersion = "1.0.0"
 )
```

Comment:
Is there another real (non-test) constant for this elsewhere that should be used ?

Reply:
Originally I made this its own constant in the test code specifically so that if we change the "real" version (CurrentConfigVersion in common/config/eventingkafkaconfig.go) without paying attention to the test, then the test will fail.  However ... the upgrade code became less complicated and the tests that used it aren't there, so now it's only in TestNewController(), which should be using the common constant.  Will fix.